index.html

```diff
@@ -144,9 +143,8 @@ <h3>Demos:</h3>
 </button>
 <code>macOSNotif({sounds:true})</code><br/>
 
-<script src="plugin/macOSNotif.js"></script>
```

Comment:
This doesn't appear to have been replaced?

Reply:
Just noticed your comment about this, is there a way to get this working or not? On the package site it serves this page, so having it as a working demo would be nice.